src/main/java/testpackage/secondTry/subscribe/Subscribe.java

```diff
@@ -0,0 +1,6 @@
+package secondTry.subscribe;
```

Comment:
package name does not correspond to actual placement of file (should be package testpackage.secondTry.subscribe)

Reply:
Done